<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>865</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>321</width>
      <height>681</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="generateGraphLbl">
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Генерация графа</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QLineEdit" name="vertexGenerateEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="densityLbl">
         <property name="text">
          <string>Плотность, %</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="densityEdit"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="vertexCountLbl_2">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Количество вершин</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="isDirectedCBox">
       <property name="text">
        <string>Ориентированный</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="weightedCBox">
       <property name="text">
        <string>Взвешенный</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="OneDirectCB">
       <property name="text">
        <string>Однонаправленный</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Диапазон весов:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>от</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="minWeightEdit">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>до</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="maxWeightEdit">
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="generateBtn">
       <property name="text">
        <string>Генерация графа</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="edgesInputTable">
       <property name="rowCount">
        <number>1</number>
       </property>
       <property name="columnCount">
        <number>3</number>
       </property>
       <row/>
       <column/>
       <column/>
       <column/>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SaveBtn">
       <property name="text">
        <string>Сохранить в файл</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="OpenBtn">
       <property name="text">
        <string>Загрузить из файла</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ACOBtn">
       <property name="text">
        <string>ACO</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="GABtn">
       <property name="text">
        <string>GA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowser"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
